packages/auth/amplify_auth_cognito/example/integration_test/hosted_ui_webview_test.dart

```diff
@@ -0,0 +1,256 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// ignore_for_file: invalid_use_of_protected_member
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:amplify_auth_cognito/amplify_auth_cognito.dart';
+// ignore: implementation_imports
+import 'package:amplify_auth_cognito/src/flows/hosted_ui/hosted_ui_platform_flutter.dart';
+import 'package:amplify_auth_cognito_example/amplifyconfiguration.dart';
+import 'package:amplify_flutter/amplify_flutter.dart';
+import 'package:amplify_test/amplify_test.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:webview_flutter/webview_flutter.dart';
+// ignore: implementation_imports
+import 'package:webview_flutter_wkwebview/src/foundation/foundation.dart';
+
+import 'utils/setup_utils.dart';
+
+late String username;
```

Comment:
Are these top level so that HostedUiTestPlatform has access? You could probably pass them into the constructor for HostedUiTestPlatform

Reply:
True. I can change it